### ISSUE TITLE
Dependabot is no longer working

### ISSUE THREAD

**Issue by nicojs** [stryker-mutator/stryker-js#3500]
**Is your maintenance request related to a problem? Please describe.**
I'm pretty frustrated by this. Dependabot isn't well suited for monorepos.

See https://github.com/dependabot/dependabot-core/issues/4364#issuecomment-1038194085 and https://github.com/dependabot/dependabot-core/issues/4364#issuecomment-1118256638

**Describe the solution you'd like**
We should either:
1. make dependabot work,
2. or migrate to the [renovatebot gh action](https://github.com/renovatebot/github-action)
3. Or implement something ourselves using [`npm-check-updates`](https://www.npmjs.com/package/npm-check-updates)

**Describe alternatives you've considered**
See above.

**Additional context**
Stryker is not supporting Jest@28, see #3481. Only seen because users migrated to 28 and reported this. Dependabot was ... missing in action

**Comment by pmcelhaney** [comment]
That's strange. I discovered it _because_ of Dependabot on my repo. I'll have a look at your set up and see if I can help.

**Comment by pmcelhaney** [comment]
Okay, I've caught up in the other thread. It's not pretty but I wonder if this will work. 

```yaml

updates:
  - package-ecosystem: npm
    directory: '/packages/core'
  - package-ecosystem: npm
    directory: '/packages/util'
  - ...

```

**Comment by pmcelhaney** [comment]
Yeah, that’s not a solution, it’s a short term workaround. 

Renovate looks cool but I wonder what’s stopping people from fixing Dependabot? I might give it a shot myself.

**Comment by pmcelhaney** [comment]
Ugh, 20 minutes and counting to run Dependabot's unit tests on an M1 Pro Max. 

The [set up for Renovate](https://docs.renovatebot.com/getting-started/installing-onboarding/) looks pretty simple but an admin has to install it.

**Comment by nicojs** [comment]
If you want: I'm on vacation right now, but I will activate renovate when I get back. In the mean time, feel free to create a PR for the renovate config (and removal of dependabot config).

I will add the secrets under `secrets.RENOVATE_TOKEN` and `secrets.RENOVATE_GH_TOKEN`.

**Comment by nicojs** [comment]
Fixed with #3512
